docs/source/algorithms/FindDetectorsOutsideLimits-v1.rst

```diff
@@ -41,15 +41,15 @@ Usage
         ws.setY(i,np.array(deadDetArray))
         ws.setY(i+1,np.array(noisyDetArray))
 
-    print "With just the default LowThreshold of 0"
+    print("With just the default LowThreshold of 0")
     (wsOut,NumberOfFailures)=FindDetectorsOutsideLimits(ws)
-    print "%i spectra were outside the limits." % NumberOfFailures
-    print
+    print("{} spectra were outside the limits.".format(NumberOfFailures))
+    print("")
```

Comment:
You could of just done `print()` here and one other place.

Reply:
`print()` prints `()` in Python 2 - I kept it this way for backwards compatibility

Reply:
🤦‍♂️ 👍